Sort and limit highlight video

runnersaw opened the pull request:
Needs to go in after this change:

https://bitbucket.org/santosfamilyfoundation/trafficintelligence/pull-requests/new?source=include-ttc&t=1

runnersaw committed:
Sort and limit highlight video

dcelik commented on app/handlers/createHighlightVideo.py:
```diff
@@ -83,6 +83,11 @@ def handler(identifier, email, ttc_threshold, vehicle_only):
             return (500, str(error_message))
 
+        near_misses = sorted(near_misses, key=(lambda x: x[4]))
+        num_near_misses_to_use = 5
```
This should be a default argument we pass in so that the user can change the number they want to see, much like num_frames in test_config.

runnersaw replied:
Sure, but we still should discuss a maximum value for this. I'd like to propose a semi-arbitrary 10?

dcelik replied:
That sounds good, I think 10 is a fine number for now.

Commit:
Update: highlight limit to match TI

Summary:
- Removed ordering by severity here, since its done by TI
- Added num_near_misses_to_use as an argument
- Set max limit to 10

dcelik reviewed (approved):
Looks good, nice conversion to SQL instead of python sorting.

youralien commented:
This was the commit in Traffic Intelligence this depends on, fyi:

https://bitbucket.org/santosfamilyfoundation/trafficintelligence/pull-requests/3/limit-and-order-near-miss-by-severity/diff
